src/web_interface/components/io_routes.py

```diff
@@ -52,10 +56,12 @@ def _download_base64_decoded_section(self, uid, section, expression_id):
             binary = binascii.a2b_base64(binary[span_in_section[0]:span_in_section[1]])
         except binascii.Error as error:
             return render_template('error.html', message=str(error))
-        else:
-            resp = make_response(binary)
-            resp.headers['Content-Disposition'] = 'attachment; filename={}'.format(file_obj.file_name + '_0x%X' % (span_in_binary[0] + span_in_section[0]) + '-0x%X_decoded' % (span_in_binary[1] - span_in_section[2]))
-            return resp
+        response = make_response(binary)
+        file_name = (file_obj.file_name
```

Comment:
original nasty code seems to come from me, still..
Would it not conform to our string format style if the string + string + string part is rewritten to something like `'{}{}{}'.format(string, string, string)`?

Reply:
I just can't wait for the day we adapt f-strings...